src/CP/core/fixPoint.jl

```diff
@@ -43,5 +43,20 @@ function fixPoint!(model::CPModel, new_constraints::Union{Array{Constraint}, Not
         end
     end
 
+    if isnothing(model.statistics.searchTreeSize)
+        model.statistics.searchTreeSize = SeaPearl.computeSearchTreeSize!(model)

```

Comment:
The estimated search tree is computed after each fix-point, and used to compute the impact for the last assignation variable/value.